edx_proctoring/admin.py

```diff
@@ -334,6 +343,255 @@ def save_model(self, request, review, form, change):
+        )
+
+    def queryset(self, request, queryset):
+        """
+        Return the filtered queryset
+        """
+
+        if self.value() in [ProctoredExamStudentAttemptStatus.created, ProctoredExamStudentAttemptStatus.submitted]:
+            return queryset.filter(status=self.value())
+        else:
+            return queryset
+
+
+class ExamAttemptFilterIsProctored(admin.SimpleListFilter):
+    """
+    Quick filter to allow admins to see attempts by flag "taking_as_proctored"
+    """
+
+    title = _('Taking as Proctored')
+    parameter_name = 'taking_as_proctored'
+
+    def lookups(self, request, model_admin):
+        """
+        List of values to allow admin to select
+        """
+        return (
+            ('True', _('True')),
+            ('False', _('False')),
+        )
+
+    def queryset(self, request, queryset):
+        """
+        Return the filtered queryset
+        """
+        if self.value() == 'True':
+            return queryset.filter(taking_as_proctored=True)
+        elif self.value() == 'False':
+            return queryset.filter(taking_as_proctored=False)
+        else:
+            return queryset
+
+
+class ExamAttemptFilterIsSample(admin.SimpleListFilter):
+    """
+    Quick filter to allow admins to see attempts by flag "is_sample_attempt"
+    """
+
+    title = _('Is Sample Attempt')
+    parameter_name = 'is_sample_attempt'
+
+    def lookups(self, request, model_admin):
+        """
+        List of values to allow admin to select
+        """
+        return (
+            ('True', _('True')),
+            ('False', _('False')),
+        )
+
+    def queryset(self, request, queryset):
+        """
+        Return the filtered queryset
+        """
+        if self.value() == 'True':
+            return queryset.filter(is_sample_attempt=True)
+        elif self.value() == 'False':
+            return queryset.filter(is_sample_attempt=False)
+        else:
+            return queryset
+
+
+class ExamAttemptFilterByCourseId(admin.SimpleListFilter):
+    """
+    Quick filter to allow admins to see attempts by "course_id"
+    """
+
+    title = _('Course Id')
+    parameter_name = 'proctored_exam__course_id'
+
+    def lookups(self, request, model_admin):
+        """
+        List of values to allow admin to select
+        """
+        attempts = ProctoredExamStudentAttempt.objects.all()
+
+        lookups = (())
+
+        unique_course_ids = []
+        for attempt in attempts:
+            course_id = attempt.proctored_exam.course_id.replace('+', ' ').replace('/', ' ').replace('course-v1:', '')
+            if course_id not in unique_course_ids:
+                unique_course_ids.append(course_id)
+                lookups += ((
+                    attempt.proctored_exam.course_id,
+                    u'{course_id}: {exam_name}'.format(
+                        course_id=course_id,
+                        exam_name=attempt.proctored_exam.exam_name
```

Comment:
there can be multiple exams per course, so I think you just need to format course_id into the display string